deploy.sh

```diff
@@ -1,3 +1,31 @@
 #!/bin/bash -e
 
-rsync -e ssh -p 2222 -avz --exclude-from rsync-exclude-list.txt ./ USERNAME@DOMAIN:FOLDER
+# CHOOSE YOUR DEPLOYMENT CONFIGURATION
+# Below are a few options for you to deploy your code
+# Choose one by commenting out the others
+# Or comment them all out and add your own
+
+
+# All variables hardcoded
+# Hardcode your  username@server.name:directory (ex. johndoe@mysite.com:public-www)
+# rsync -e ssh -p -avz --exclude-from deploy-exclude-list.txt ./_site/ USERNAME@SERVER.NAME:DIRECTORY
+
+
+# Prompt for username
+# Hardcode your server.name:directory (ex. mysite.com:public-www)
+# echo "Please enter username: "
+# read usr
+# rsync -e ssh -p -avz --exclude-from deploy-exclude-list.txt ./_site/ $usr@SERVER.NAME:DIRECTORY
+
+
+# Prompt for each detail
+echo "Please enter username"
+read usr
+echo "Please enter server.name:diretory"
```

Comment:
No, but some people are overly cautious with their security, probably more so the directory than the server name. If you think it is overkill, I can remove it. Also, I need to spell directory correctly there! ;)